posthog/temporal/batch_exports/batch_exports.py

```diff
@@ -449,6 +476,7 @@ def __init__(self, task: str):
         super().__init__(f"Expected task '{task}' to be done by now")
 
 
+# TODO - not sure this is being used anymore?
```

Comment:
Nope, should be fine to remove.

Reply:
cool, have removed it. Think there's quite a lot of other code that can be removed from this file too but will save that for a future PR